source/config/mockData/index.js

```diff
@@ -0,0 +1,20 @@
+//Packages
+import { createServer } from "miragejs"
+
+//scripts
+import routes from './routes.json';
+import loginInfo from './data/security-login.json'
+import URL from '../../utils/config'
+
+export default createServer({
+  routes() {
+    this.post(`${URL}${routes.login}`, (schema, request) => {
```

Comment:
See how we setup config.services in Celerik projects

This should be like this.post(config.services.security.login, callback)